cli/main.py

```diff
@@ -0,0 +1,135 @@
+from task_list import *
+
+OPTIONS = {
+        "1": "List all tasks", 
+        "2": "Create a task",
+        "3": "View one task", 
+        "4": "Update task", 
+        "5": "Delete task", 
+        "6": "Mark complete",
+        "7": "Mark incomplete",
+        "8": "Delete all tasks",
+        "9": "List all options",
+        "10": "Quit"
+        }
+
+def list_options():
+
+    for number, feature in OPTIONS.items():
+        print(f"{number}. {feature}")
+
+
+def make_choice():
+    valid_choices = OPTIONS.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        print("\n What would you like to do? ")
+        choice = input("Make your selection using the option number. Select 9 to list all options: ")
+
+    return choice
+
+def get_task_from_user(msg = "Input the id of the task you would like to work with: "):
+    id = input(msg)
+    task = get_task(id)
+    if not task:
+        print_surround_stars("I cannot find that task.")
+    return task
```

Comment:
I think this is fine, but it could be restructured it so it's more similar to the make_choice function and I added some error checking (and some ui tweaks for fun):
```suggestion
    task = None
    tasks = list_tasks()
    if not tasks:
        print_stars("This option is not possible because there are no tasks.")
        return task
    count = 0
    help_count = 3 #number of tries before offering assistance
    while not task:
        id = input(msg)
        task = get_task(id)
        if not task:
            print_surround_stars("I cannot find that task.  Please try again.")
        count += 1
        if count >= help_count:
            print("You seem to be having trouble selecting a task.  Please choose from the following list of tasks.")
            print_all_tasks()
        
    return task
```